ape_alchemy/providers.py

```diff
@@ -85,7 +85,7 @@ def send_transaction(self, txn: TransactionAPI) -> ReceiptAPI:
         except ValueError as err:
             raise _get_vm_error(err) from err
 
-        receipt.raise_for_status(txn)
```

Comment:
was a bug due to a change in ape a while back, this should have been updated before but wasn't caught except by `mypy`